G1ANT.Addon.IExplorer/G1ANT.Addon.IE.Tests/Commands/ie.open/ShouldGetTitle_WhenOpenMultipleGoogle.G1ANT

```diff
@@ -0,0 +1,20 @@
+﻿addon iexplorer version 2.205.17304.1000
+addon core version 4.99.18340.1354
+addon language version 4.99.18340.1354
+
+ie.open ‴www.google.pl‴ result ♥IE_Handle1
```

Comment:
Don't like the numbers here. 1 => google, 2 => g1ant, what for is a case #3?

Reply:
there are 3 ways to execute ie.open
ie.open ‴www.google.pl‴
ie.open url ‴https://g1ant.com‴
ie.open <<this one just open ie with no specific page to load